src/views/preview/remix-list.jsx

```diff
@@ -9,9 +9,9 @@ const RemixList = props => {
     if (remixes.length === 0) return null;
     return (
         <FlexRow className="remix-list">
-            <div className="project-title">
+            <h2>
                 Remixes
-            </div>
+            </h2>
```

Comment:
Please use a specific class name like it was doing before instead of relying on the style of `h2`. Very soon we will need to stop using `h1`, `h2`, etc. everywhere and switch to this method instead, so we may as well not add more debt.

Reply:
Ah, switching to a class name will make this fix way more secure, good idea!

Reply:
(h1, h2 and so on tags might be important for screenreaders?)

Reply:
(h1, h2 and so on might be important for screen readers?)